unused/RemovedUi.java

```diff
@@ -0,0 +1,31 @@
+//@@author PraveenElango
```

Comment:
What is the purpose of this file?

Reply:
Oh, I was browsing through the CS2113T site today and came across this - https://nus-cs2113-ay2021s1.github.io/website/admin/tp-w10.html#4-make-the-code-reposense-compatible. It is mentioned under the RepoSense section that we can include code individually written that is not used in the final product by placing it in the unused folder.

Reply:
Ohh  I see I see

Reply:
BTW did you test how the commands for boolean look with the jar file in a terminal
Longer lines are generally pushed to the next line thereby resulting in inconsistent tabs. So consider adding tabs and newline wherever necessary after checking. I made changes for circuit already (probably going to have merge conflict with your  current changes of circuit commands lol)

Reply:
I didn't test with jar file yet but I did add tabs and spaces when the sentences were too long for those in the Boolean tutorial. I'll test in the jar file and get back to you.